src/applications/personalization/profile-2/components/connected-apps/ConnectedApps.jsx

```diff
@@ -82,14 +87,15 @@ export class ConnectedApps extends Component {
           </>
         )}
 
-        {!isEmpty(errors) && (
-          <AlertBox
-            className="vads-u-margin-bottom--2"
-            headline="We couldn’t retrieve your connected apps"
-            status="warning"
-            content="We’re sorry. Something went wrong on our end and we couldn’t access your connected apps. Please try again later."
-          />
-        )}
+        {!isEmpty(errors) &&
+          !recordNotFound && (
```

Comment:
This would be a little nicer if this logic was wrapped up in another boolean like `showGeneralError`

Reply:
Done!